schema/model.json

```diff
@@ -12,6 +14,8 @@
       },
 
       "resources": {
+        "modelversion": "0.5",
+        "compatiblewith": "https://raw.githubusercontent.com/xregistry/spec/refs/heads/main/schema/model.json",
```

Comment:
these need to be under a resource (e.g. "schemas") not the "resources" map.
This will fix the validation error you're seeing

Reply:
Fixed it, thanks!